DEVELOPMENT.md

```diff
@@ -72,6 +73,8 @@ recommend adding them to your `.bashrc`):
 1. `KO_DOCKER_REPO`: The docker repository to which developer images should be
    pushed (e.g. `gcr.io/[gcloud-project]`).
 
+- **Note:** skip the go environment varialbles steps (1&2) if you have installed
```

Comment:
```suggestion
- **Note:** skip the go environment variables steps (1&2) if you have installed
```

Reply:
(.. though, fwiw Im a bit torn about whether this line is really needed at all: doesn't the fact the bullet already says "If you don't have one" cover this?)